### PR TITLE
Resolve empty queues immediately

### PR DESCRIPTION
**Pull request by rich-nguyen**

### CODE REVIEW

File: static/src/javascripts/projects/common/modules/discussion/whole-discussion.js [hunk 1]
```diff
@@ -38,6 +38,11 @@ define([
                 workFunction.call(null, item).then(onComplete, onComplete);
             }
 
+            if (!items || !items.length) {
+                resolve();
+                return;
```
Comment (phamann) [inline static/src/javascripts/projects/common/modules/discussion/whole-discussion.js]: No need for the return statement

**Comment by phamann** [comment]
:+1:

File: static/src/javascripts/projects/common/modules/discussion/whole-discussion.js (hunk 1 above)
Comment (phamann) [inline static/src/javascripts/projects/common/modules/discussion/whole-discussion.js]: Ignore me.

**Comment by mattosborn** [comment]
:+1:

**Pull request merged**